package_publisher/core.py

```diff
@@ -20,8 +23,11 @@ def upload_package(self, file_path: str, provenance_bundle_path: str) -> Any:
         ]
         command += ["--form", "file=@{}".format(file_path)]
         command += (
-            ["--form", "provenance_bundle=@{}".format(provenance_bundle_path)]
-            if provenance_bundle_path != ""
+            [
+                "--form",
+                "attestation_bundle=@{}".format(self.attestation_bundle_path),
+            ]
```

Comment:
@phantomwhale I'm calling the new attestation bundle file param `attestation_bundle` for now - reasonable?